modules/33acrossBidAdapter.js

```diff
@@ -342,6 +342,12 @@ function _buildImpORTB(bidRequest) {
     }
   };
 
+  const gpid = deepAccess(bidRequest, 'ortb2Imp.ext.gpid');
+
+  if (gpid) {
+    imp.ext.gpid = gpid;
```

Comment:
My recommendation here is to make the solution less verbose, this is pure esthetic preference, you don't have to make this change if you don't want to..
e.g.
```
const gpid = deepAccess(bidRequest, 'ortb2Imp.ext.gpid');
  const imp = {
    id: bidRequest.bidId,
    ext: {
      ttx: {
        prod: deepAccess(bidRequest, 'params.productId')
      },
      ...(gpid ? { gpid } : {})
    }
  };
```

in the example above I avoid the `if` statement and having to repeat `imp.ext`

Reply:
For gpid ? { gpid } : {})
Just want to make sure I'm understanding- if gpid is not defined, would this still set the gpid field to {}?
I was thinking it would be better to only set gpid field if there's a value defined - this way demand partners don't see the field if there's no gpid value (in case partners see {} as a valid gpid)

Reply:
no, it won't be set. `(gpid ? { gpid } : {})` will evaluate to `({ })` and the destructions of `...({ })` won't add any new fields.

Reply:
thanks! I made the change for this